packages/adaptive-ui-designer-figma/src/cli/library-schema.ts

```diff
@@ -0,0 +1,19 @@
+export default {
+  $schema: 'http://json-schema.org/draft-07/schema#',
+  $id: 'https://highspot.com/schemas/design-to-code-library.json',
```

Comment:
We don't have a registered domain name yet, but should probably use something else here.

Reply:
Good catch. This just needs some valid URL so I put it in the adaptive-web-community domain.